Aztec/Classes/Public/Extensions/CGRect+Helpers.swift

```diff
@@ -0,0 +1,56 @@
+import CoreGraphics
+
+extension CGRect {
+    /// The position to which a rect is anchored when resized vertically.
+    ///
+    enum VerticalAnchor {
+        case top
+        case middle
+        case bottom
+    }
+
+    /// The position to which a rect is anchored when resized horizontally.
+    ///
+    enum HorizontalAnchor {
+        case left
+        case center
+        case right
+    }
+
+    /// Return a rect resized to a specific size, while anchoring it to one of
+    /// their edges or centers.
+    ///
+    /// By default, changing a CGRect size will resize it anchored to its top
```

Comment:
> CGRect is typically used in UIKit so the origin is in the upper-left corner and the rectangle extends towards the lower-right corner. This method adds the option to grow or shrink the rect anchoring to other points.

I think the first part can be removed completely, since it talks about something that's really tangential to the scope of this method.  In other words, this is an extension to (a dumb struct named) CGRect adding some support for resizing + anchoring.  This support is offered in a really broad and generic way, non specific to UIKit.

I would maintain (a slightly modified version of) the second part and make it just:

> This method adds the option to grow or shrink the rect while anchoring it to a point.